lib/IDE/TypeReconstruction.cpp

```diff
@@ -1347,25 +1347,6 @@ static void VisitNodeFunction(
     }
   } while (0);
 
-  //                    if (node_kind == Demangle::Node::Kind::Allocator)
-  //                    {
-  //                        // For allocators we don't have an identifier for
-  //                        the name, we will
-  //                        // need to extract it from the class or struct in
-  //                        "identifier_result"
-  //                        //Find
-  //                        if (identifier_result.HasSingleType())
-  //                        {
-  //                            // This contains the class or struct
-  //                            StringRef init_name("init");
-  //
-  //                            if (FindFirstNamedDeclWithKind(ast, init_name,
-  //                            DeclKind::Constructor, identifier_result))
-  //                            {
-  //                            }
-  //                        }
-  //                    }
```

Comment:
Did you mean to leave this commented out?

Reply:
Yes, because this was wrong code (in the wrong place). I decided to remove it altogether but happy to split it in a separate commit if you prefer.

Reply:
Oh, sorry, my brain thought this was an *addition* of commented-out code. I agree that it should be removed.